doc/AUTHORS.md

```diff
@@ -166,3 +166,11 @@ documentation, Python bindings for elektraMerge, copy-on-write functionality of
 - email: max@maxirlinger.at
 - GitHub user: [atmaxinger](https://github.com/atmaxinger)
 - devel/test on: Fedora, macOS
+
+## Tomislav Makar
+
+documentation, go-elektra (spec globbing use-case), rewrite spec plugin
```

Comment:
Please add link to go-elektra

```suggestion
spec plugin, go-elektra, web-ui
```

Reply:
@markus2330 should I separate `go-elektra` and `web-ui` here, I thought of putting the web-ui (actually a web-api) into the `go-elektra` repo? What do you think?

Reply:
@markus2330 and I would call it `web-api` as it will be an api and no graphical user interface.

Reply:
You simply write in src/tools/elektrad/README.md etc. that you are the maintainer (infos/maintainer). Please use a header like we already have for plugins and bindings.

@atmaxinger please do the coordination about this and start with a PR that shows how to do it properly.